apis/supernova/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "devDependencies": {
     "extend": "3.0.2",
-    "node-event-emitter": "0.0.1"
+    "node-event-emitter": "0.0.1",
+    "@nebula.js/nucleus": "^5.12.4"
```

Comment:
this would make it a dependency loop, need to change how this is setup. Move the whole hypercube handler stuff into the supernova package instead?

Reply:
ok, I'll try it.